sieve_of_eratosthenes/sieve_of_eratosthenes.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+        "fmt"
```

Comment:
Try using the GoSublime Sublime Text plugin. You might have some problem configuring it, so here is my GoSublime user settings file.
https://gist.github.com/aviaryan/3e2ac784609e046441ee9ba9032baf17